rpmconf/Install/Util/utilfunc.sh

```diff
@@ -2484,7 +2484,7 @@ isOnlyofficeStandalone() {
   fi
   ## install rabbit mq
   if [ $onlyofficepkg == "yes" ]; then
-    INSTALL_PACKAGES="$INSTALL_PACKAGES rabbit-mq"
+    INSTALL_PACKAGES="$INSTALL_PACKAGES zimbra-rabbitmq-server"
```

Comment:
Approved for now, but in future we should remove this change and add zimbra-rabbitmq-server as a dependency of zimbra-onlyoffice package so we don't have to install it separately